src/storage/index/LookupBaseProcessor.inl

```diff
@@ -126,6 +129,10 @@ bool LookupBaseProcessor<REQ, RESP>::isOutsideIndex(Expression* filter,
  *              |  AggregateNode   |
  *              +--------+---------+
  *                       |
+ *              +--------+---------+
+ *              |    DeDupNode     |
```

Comment:
Need option in request to choose does dedup.

Reply:
yeah, I will add the distinct flag of thrift-api for lookup in the future.